modules/tinymce/src/core/main/ts/bookmark/ResolveBookmark.ts

```diff
@@ -25,16 +33,14 @@ const addBogus = (dom: DOMUtils, node: Node): Node => {
 };
 
 const resolveCaretPositionBookmark = (dom: DOMUtils, bookmark: StringPathBookmark) => {
-  let pos;
-
   const rng = dom.createRng();
-  pos = CaretBookmark.resolve(dom.getRoot(), bookmark.start);
-  rng.setStart(pos.container(), pos.offset());
+  const spos = CaretBookmark.resolve(dom.getRoot(), bookmark.start);
```

Comment:
For better readability, could use `sPos` or `startPos` as the variable name instead? Same for `epos` name as well

Reply:
Yeah I'd agree with Luke that these should be using camel case at the very least. That said, I do like `startPos` as it's clear 🤷 

- [ ] Fix naming please